### PR TITLE
feat: Add more providers

### PR DESCRIPTION
**Pull request by kushthedude**
@leecalcote

### CODE REVIEW

**Comment by leecalcote** [comment]
@Nikhil-Ladha we're adding a few more providers - disable for now.

@prateeksahu we're adding an inactive "UT Austin" provider, which serves to indicate that a repository of research data is forthcoming.

File: provider-ui/components/ProviderComponent.js [hunk 1]
```diff
@@ -158,108 +158,136 @@ class ProviderComponent extends React.Component {
...
+          </React.Fragment>
+          );                  
+      })}
+      <ListItem
+      key="SMI Tool"
+      className={classes.providerDesc}
+      >
+      Provider: SMI Tool<br />
+      <ul><li>Provider for SMI Conformance Testing</li>
+      <li>Specially create for W/G SMI Tool</li>
```
Comment (Nikhil-Ladha) [inline provider-ui/components/ProviderComponent.js:199]: Typo: *created

File: provider-ui/components/ProviderComponent.js [hunk 2]
```diff
@@ -158,108 +158,136 @@ class ProviderComponent extends React.Component {
...
+          {Object.keys(availableProviders).map((key) => (
+            <MenuItem
+            key={key}
+            onClick={() => self.handleMenuItemClick(key)}
+            >
+            {key}
+            </MenuItem>
+
+            ))}
+          <MenuItem disabled={true}>
```
Comment (Nikhil-Ladha) [inline provider-ui/components/ProviderComponent.js:267]: Please add a key here as well, even though it's disabled.

File: provider-ui/components/ProviderComponent.js [hunk 3]
```diff
@@ -158,108 +158,136 @@ class ProviderComponent extends React.Component {
...
+          SMI Tool
+          </MenuItem>
+          <MenuItem disabled={true}>
```
Comment (Nikhil-Ladha) [inline provider-ui/components/ProviderComponent.js:270]: Same as above

File: provider-ui/components/ProviderComponent.js (hunk 1 above)
Comment (leecalcote) [reply provider-ui/components/ProviderComponent.js]: Updated text.

File: provider-ui/components/ProviderComponent.js (hunk 2 above)
Comment (leecalcote) [reply provider-ui/components/ProviderComponent.js]: Added MenuItem key

File: provider-ui/components/ProviderComponent.js (hunk 3 above)
Comment (leecalcote) [reply provider-ui/components/ProviderComponent.js]: added `key="SMI"`

**Comment by Nikhil-Ladha** [comment]
Coool, Good to go!👍🏼

**Comment by leecalcote** [comment]
@prateeksahu @pranavk2 @tiwari @ame505
Screen Shot 2020-08-06 at 12 51 32 AM
Screen Shot 2020-08-06 at 12 51 09 AM

**Pull request merged**